src/views/Users.vue

```diff
@@ -13,16 +13,16 @@ export default {
   data() {
     return {
       users: [],
-    }
+    };
   },
   async created() {
     try {
       const response = await fetchUsers();
       this.users = response;
     } catch (error) {
-      showNotification('Failed to fetch users: ' + error, 'error')
+      showNotification("Failed to fetch users: " + error, "error");
     }
-  }
```

Comment:
Can we remove the error handling from the Users view as well. The error handling and notification is in the fetchUsers method itself.